test/ClientShutdownTest.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+
+var chai = require('chai');
+var expect = chai.expect;
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+var sinon = require('sinon');
+var sandbox = sinon.createSandbox();
+
+var RC = require('./RC');
+var HzErrors = require('../.').HazelcastErrors;
+var {Client} = require('../.');
```

Comment:
I got failed CI build as Node.js 4 doesn't support destructuring assignments. Could you replace them with direct assignments?

Reply:
Ok

Reply:
Ready